Gemfile

```diff
@@ -19,7 +19,7 @@ gem "rails", rails
 if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.2.0')
   gem "test-unit", "~> 3.0"
 end
-
+gem 'sprockets-rails', '~> 2.0'
```

Comment:
Sprockets 3 requires a manifest, pegged sprockets to 2 for duration of this branch so the manifest can be fixed separately.